module_utils/network_lsr/argument_validator.py

```diff
@@ -584,7 +585,10 @@ def __init__(self):
         ArgValidatorDict.__init__(
             self,
             name="ethtool",
-            nested=[ArgValidator_DictEthtoolFeatures()],
+            nested=[
+                ArgValidator_DictEthtoolFeatures(),
+                ArgValidator_DictEthtoolSetring()
```

Comment:
I think the correct name is `ArgValidator_DictEthtoolSetRing()` according to the naming use on the project. Could you confirm me this? @tyll 

Thanks!

Reply:
Actually, it seems to me that `ArgValidator_DictEthtoolRing` would be the correct name.

Reply:
Changed into `ArgValidator_DictEthtoolRing`.